### ISSUE TITLE
Wrong version number used in documentation title

### ISSUE THREAD

**Issue by Flimm** [pennersr/django-allauth#3234]
Take this link: https://django-allauth.readthedocs.io/en/latest/release-notes.html

The title of the page is "Release Notes - django-allauth 0.43.0 documentation". This is the wrong version number. The latest release is in fact 0.52.0.

The title of the page is found in the `<title>` HTML element, and its visible in Google search results, and in the user interface of the browser.

To fix it, the version needs to be changed [in the file `docs/conf.py` in these lines](https://github.com/pennersr/django-allauth/blob/30a3473b8010264143f031bb7565eb26faa143e4/docs/conf.py#L48):

```python
# The short X.Y version.
version = "0.43"
# The full version, including alpha/beta/rc tags.
release = "0.43.0"
```

**Comment by pennersr** [comment]
Fixed for 0.55.0

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3234
